daemon/daemon_unix.go

```diff
@@ -881,20 +892,17 @@ func (daemon *Daemon) initNetworkController(activeSandboxes map[string]interface
 	if !conf.DisableBridge {
 		// Initialize default driver "bridge"
 		if err := initBridgeDriver(controller, conf); err != nil {
-			return nil, err
+			return err
 		}
 	} else {
 		removeDefaultBridgeInterface()
 	}
 
-	// Set HostGatewayIP to the default bridge's IP  if it is empty
-	setHostGatewayIP(conf, controller)
-
-	return controller, nil
+	return nil
 }
 
 // setHostGatewayIP sets cfg.HostGatewayIP to the default bridge's IP if it is empty.
-func setHostGatewayIP(config *config.Config, controller libnetwork.NetworkController) {
+func setHostGatewayIP(controller libnetwork.NetworkController, config *config.Config) {
```

Comment:
Just swapping the order of arguments to match all other functions that accept `controller + config`